kernel/src/actions/mod.rs

```diff
@@ -1974,4 +1993,36 @@ mod tests {
         assert!(record_batch.column(2).is_null(0));
         assert!(record_batch.column(3).is_null(0));
     }
+
+    #[test]
```

Comment:
for completeness should probably check that it works if you have `ADD_NAME` and also something that's _not_ a file action. :)